packages/python/m/cli/commands/github/branch_prs.py

```diff
@@ -20,14 +20,6 @@ class Arguments(BaseModel):
         default=False,
         description='use yaml format',
     )
-    owner: str = Field(
-        default=env('GITHUB_REPOSITORY_OWNER'),
```

Comment:
Lets write an entry on the CHANGELOG that github commands will no longer use this variable since by default the owner will be read from the git configuration where the m cli is run.